src/config.ts

```diff
@@ -268,21 +268,18 @@ export const CLIENT_OPTIONS: ClientOptions = {
 		'DIRECT_MESSAGE_REACTIONS'
 	],
 	loadDefaultErrorListeners: false,
-	makeCache: Options.cacheWithLimits({
-		...Options.defaultMakeCacheSettings,
-		MessageManager: {
-			sweepInterval: 180,
-			sweepFilter: LimitedCollection.filterByLifetime({
-				lifetime: 900,
-				getComparisonTimestamp: (m) => m.editedTimestamp ?? m.createdTimestamp
-			})
+	sweepers: {
```

Comment:
```suggestion
  makeCache: {}, // or MessageManager: 200 if you wanted to keep that
	sweepers: {
```

Need to override makeCache still since I can't change default options in semver minor 😢

Reply:
Thanks for checking this out 😅

Reply:
mmhmm, sorry I broke lint lol, didn't notice you were using tabs

Reply:
Easy fix:
```suggestion
	makeCache: {},
	sweepers: {
```

Reply:
uh...I think that broke it more lol

Reply:
also interesting ts error you got

Reply:
Unbroke it, lol